src/EventListener/ApiExceptionListener.php

```diff
@@ -21,21 +35,29 @@ public function onKernelException(GetResponseForExceptionEvent $event): void
         $exception = $event->getException();
 
         if ($exception instanceof HttpExceptionInterface) {
+            $data = [
+                'code' => $exception->getStatusCode(),
+                'message' => $exception->getMessage(),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "message" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)